lib/spack/spack/test/cmd/module.py

```diff
@@ -13,18 +13,16 @@
 
 module = spack.main.SpackCommand('module')
 
+#: make sure module files are generated for all the tests here
+pytestmark = pytest.mark.usefixtures('ensure_module_files_are_there')
+
 
 def _module_files(module_type, *specs):
     specs = [spack.spec.Spec(x).concretized() for x in specs]
     writer_cls = spack.modules.module_types[module_type]
     return [writer_cls(spec).layout.filename for spec in specs]
 
 
-@pytest.fixture(scope='module', autouse=True)
-def ensure_module_files_are_there(database):
-    module('tcl', 'refresh', '-y')
```

Comment:
This fixture is only used here. Can we find a way to use the new context managers without pushing the fixture into the main `conftest.py`?

Reply:
done